src/style/app.module.css

```diff
@@ -7251,3 +7251,48 @@ button[data-testid='createPostBtn'] {
   flex-direction: row;
   justify-content: center;
 }
+
+.alignimg {
+  border-radius: 50%;
+  background-color: var(--white-color);
+  background-blend-mode: darken;
+  height: clamp(50px, 4vw, 65px);
+  width: clamp(50px, 4vw, 65px);
+  object-fit: cover;
+}
+
+.box {
+  color: var(--light-orange);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`